drake/systems/plants/collision/ResultCollector.cpp

```diff
@@ -0,0 +1,19 @@
+#include <iostream>
+
+#include "drake/systems/plants/collision/DrakeCollision.h"
+#include "ResultCollector.h"
+
+using namespace std;
+using namespace Eigen;
+
+namespace DrakeCollision {
+Vector3d toVector3d(const Vector3d& vec) { return vec; }
```

Comment:
It is moved in the fcl branch... but doesn't seem to be used by the fcl code... If anything, I wonder if it should be moved to `bullet_model.cc` instead; it appears to be an implementation detail of that code that is not used anywhere else.